html/views/commit/commit.css

```diff
@@ -94,8 +94,9 @@ table.diff {
     background-color: #7795c7;
 }
 
-#selected #stagelines {
+#selected .hunkbutton {
     background-color: #FFF;
+	box-shadow: 0px 0px 1px #cce5ff;
```

Comment:
Why add a box-shadow? It's fine, but it's quite subtle and the other buttons don't have a glow.